pages/search/index.vue

```diff
@@ -136,8 +144,13 @@
         page: 1
       };
     },
-    asyncData ({ env, query, res }) {
-      const currentPage = query.page ? Number(query.page) : 1;
+    asyncData ({ env, query, res, redirect }) {
+      const currentPage = pageFromQuery(query.page);
+      if (currentPage === null) {
+        // Redirect non-positive integer values for `page` to `page=1`
+        return redirect({ name: 'search', query: { query: query.query || '', page: '1' } });
```

Comment:
This and the pagination links below might get messy if we continue to create them like this by plucking each value. Already thinking about adding multiple facets.
Something like:
```
        query['page'] = '1';
        return redirect({ name: 'search', query: query });
```
Will be easier to maintain for when we start adding more params.